src/main/java/seedu/address/logic/commands/KeyCommand.java

```diff
@@ -65,7 +65,8 @@ public CommandResult execute() throws CommandException {
             }
 
             logger.info("Lock state is now: " + Boolean.toString(model.getLockState()));
-            return new CommandResult(MESSAGE_SUCCESS);
+            return new CommandResult(MESSAGE_SUCCESS + "\nLock state is now: "
+                    + Boolean.toString(model.getLockState()));
```

Comment:
Information that shows the user the state of MTM after every execution of the command, this would make the user not do so many commands in order to find the state of MTM.